ipn/ipnlocal/ssh.go

```diff
@@ -119,7 +120,7 @@ func (b *LocalBackend) getSystemSSH_HostKeys() (ret []ssh.Signer, err error) {
 	for _, typ := range keyTypes {
 		filename := "/etc/ssh/ssh_host_" + typ + "_key"
 		hostKey, err := ioutil.ReadFile(filename)
```

Comment:
should we also be following symlinks?

Reply:
we already do. are you saying should we not?

Reply:
Nvm, I thought that was part of the bug. Carry on.